### PR TITLE
New Cheat Sheet: Tagalog language

### PR DESCRIPTION
**Pull request by mayank7**
New Cheat Sheet: Tagalog language

IA Page: https://duck.co/ia/view/tagalog_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Tagalog Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/tagalog_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Tagalog Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/tagalog_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Tagalog Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/tagalog_cheat_sheet_mobile.png)

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=Tagalog Cheat Sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/tagalog_cheat_sheet_mobile.png)

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "I need to practice my Tagalog",
+         "val": "Kailangan ko pang pag-aralan ang tagalog."
+      }, {
+         "key": "Don't worry!",
+         "val": "Huwag kang mag-alala."
+      }],
+      "Expressions and Words": [{
+         "key": "Good/ Bad/ So-So.",
+         "val": "Maganda/Masama"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Ayos lang/Sablay/Sakto lang`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "Nothing much",
+         "val": "Wala naman"
+      }, {
+         "key": "See you later!",
+         "val": "Sa muling pagkikita!"
+      }],
+      "Asking for Help and Directions": [{
+         "key": "I'm lost",
+         "val": "Ako’y nawawala"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Nawawala ako`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 3]
```diff
@@ -0,0 +1,288 @@
...
+         "val": "Nasaan ang (palikuran/botika)?"
+      }, {
+         "key": "Go straight! then turn left/ right!",
+         "val": "Diretso lang, tapos kaliwa/kanan!"
+      }, {
+         "key": "I'm looking for john.",
+         "val": "Hinahanap ko si John."
+      }, {
+         "key": "One moment please!",
+         "val": "Isang Saglit lang po."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Saglit lang po!`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 4]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "Hold on please! (phone)",
+         "val": "Sandali lamang po."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Sandali lang po!`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 5]
```diff
@@ -0,0 +1,288 @@
...
+         "val": "Ano ang ikinabubuhay mo?"
+      }, {
+         "key": "I work as a (translator/ businessman)",
+         "val": "Nagtatrabaho ako bilang (Translator/Negosyante)"
+      }, {
+         "key": "I like Tagalog",
+         "val": "Gusto ko ang Tagalog."
+      }, {
+         "key": "I've been learning Tagalog for 1 month",
+         "val": "Pinag-aralan ko ang tagalog ng isang buwan."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Isang buwan na akong nag aaral ng tagalog.`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 6]
```diff
@@ -0,0 +1,288 @@
...
+         "val": "Mabuti iyan!"
+      }, {
+         "key": "How old are you?",
+         "val": "ilang taon ka na?"
+      }, {
+         "key": "I'm (twenty, thirty...) years old.",
+         "val": "Ako’y (Dalawampung, tatlumpung..) taon gulang na."
+      }, {
+         "key": "I have to go",
+         "val": "Kailangan ko ng umalis."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Kailangan ko nang umalis.`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 7]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "I will be right back!",
+         "val": "Babalik ako."
+      }],
+      "Wish Someone Something": [{
+         "key": "Happy birthday!",
+         "val": "Maligayang Kaarawan!"
+      }, {
+         "key": "Happy new year!",
+         "val": "Maligayang Bagong Taon!"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Manigong Bagong Taon!`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 8]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "Merry Christmas!",
+         "val": "Maligayang Pasko!"
+      }, {
+         "key": "I'd like to visit the Philippines one day",
+         "val": "Gusto kong bisitahin ang Pilipinas isang araw."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Nais kong bumisita sa Pilipinas ng minsan sa aking buhay.`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 9]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "Say hi to John for me",
+         "val": "Ikumusta mo na lang ako kay John."
+      }, {
+         "key": "Good night and sweet dreams!",
+         "val": "Gandang Gabi!"
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Gandang gabi at balutin sana ng tamis ang iyong mga panaginip!`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 10]
```diff
@@ -0,0 +1,288 @@
...
+         "val": "Pakisulat nga po!"
+      }, {
+         "key": "I Don't Understand!",
+         "val": "Hindi ko maintindihan."
+      }, {
+         "key": "I Don't Know!",
+         "val": "Hindi ko alam."
+      }, {
+         "key": "I Have No Idea.",
+         "val": "Wala akong ideya."
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Aba, malay ko riyan.`

File: share/goodie/cheat_sheets/json/language/tagalog.json [hunk 11]
```diff
@@ -0,0 +1,288 @@
...
+      }, {
+         "key": "Good evening!",
+         "val": "Magandang Gabi!"
+      }, {
+         "key": "How are you?",
+         "val": "Kumusta?"
+      }, {
+         "key": "I'm fine, thanks!",
+         "val": "Mabuti naman,Salamat!"
+      }, {
```
Comment (tagawa) [inline share/goodie/cheat_sheets/json/language/tagalog.json]: This should be `Mabuti naman, salamat!`

**Comment by tagawa** [comment]
Added several comments for improvements, thanks to [Julian Javier on Twitter](https://twitter.com/TheMadHakker)

**Pull request closed without merging**